### PR TITLE
Conform to OT tagging for databases and message brokers

### PR DESCRIPTION
**Pull request by sm7x**

### CODE REVIEW

File: src/plugins/amqp10.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ function startReceiveSpan (tracer, config, link) {
     tags: {
       'resource.name': `receive ${source}`,
       'span.kind': 'consumer',
-      'amqp.link.source.address': source,
+      'message_bus.destination': source,
```
Comment (rmfitzpatrick) [inline src/plugins/amqp10.js]: Do you mind updating the `peer.*` tags as well?  They are currently `amqp.connection.*`

File: src/plugins/amqplib.js [hunk 2]
```diff
@@ -98,12 +98,21 @@ function addTags (channel, tracer, config, span, method, fields) {
...
+    destinationName = fields.queue
+  }
+
   span.addTags({
     'service.name': config.service || `${tracer._service}-amqp`,
     'resource.name': getResourceName(method, fields),
     'component': 'amqplib'
   })
 
+  if (destinationName) {
```
Comment (rmfitzpatrick) [inline src/plugins/amqplib.js]: nit: why is this separate from the var definition?

File: src/plugins/memcached.js [hunk 3]
```diff
@@ -35,7 +35,8 @@ function wrapQueryCompiler (original, client, server, scope, span) {
 
     span.addTags({
       'resource.name': query.type,
-      'memcached.command': query.command
+      'memcached.command': query.command,
```
Comment (rmfitzpatrick) [inline src/plugins/memcached.js]: `db.statement`?

File: src/plugins/amqp10.js (hunk 1 above)
Comment (sm7x) [reply src/plugins/amqp10.js]: They are already set elsewhere and tested here: https://github.com/signalfx/signalfx-nodejs-tracing/blob/959251116a316f33e14eccf656d395ecb195dabc/test/plugins/amqp10.spec.js#L75

So should I take out the `amqp.connection.*` tags?

File: src/plugins/amqplib.js (hunk 2 above)
Comment (sm7x) [reply src/plugins/amqplib.js]: Because I want the tag to be set after the the tags are set.

File: src/plugins/amqp10.js (hunk 1 above)
Comment (rmfitzpatrick) [reply src/plugins/amqp10.js]: Since redundant, I'd think that's ok.

Comment (sm7x) [reply src/plugins/amqp10.js]: I just looked through the instrumentation again, `peer.port` and `peer.hostname` are only set when sending messages not when consuming (and this is sufficient for inferred services). 

So the `amqp.connection.*` tag is only redundant when messages are being sent. Should I update it to `peer.*` to be in line with OT?

**Pull request merged**